src/setup/setup.ts

```diff
@@ -168,11 +168,7 @@ export function createInstance(
   }
 }
 
-function getDocument(
-  options: Partial<Config>,
-  node: Node | undefined,
-  defaults: Required<Options>,
-) {
+function getDocument(options: Options, defaults: Config, node?: Node) {
```

Comment:
It seems like `createConfig` and `getDocument` take the same arguments, but why not unify the order?